deployments/liqo/values.yaml

```diff
@@ -120,3 +119,26 @@ global:
   authServer:
     ip: ""
     port: ""
+
+advController:
+  image:
+    repository: "liqo/advertisement-operator"
+    pullPolicy: "IfNotPresent"
+
+broadcaster:
+  image:
+    repository: "liqo/advertisement-broadcaster"
+    pullPolicy: "IfNotPresent"
+
+virtualKubelet:
+  image:
+    repository: "liqo/virtual-kubelet"
+    pullPolicy: "IfNotPresent"
+
+initVk:
+  image:
+    repository: "liqo/init-vkubelet"
+    pullPolicy: "IfNotPresent"
+
+suffix: ""
+version: "latest"
```

Comment:
These two fields are already set at the beginning of the file, and there are also inside the global object

```yaml
suffix: ""
version: "latest"
...
global:
  suffix: ""
  version: ""
...
suffix: ""
version: "latest
```

I think that we can delete some of them